wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -1981,6 +1912,105 @@ WOLFSSL_LOCAL int tsip_Tls13SendCertVerify(WOLFSSL* ssl)
         }
     }
 
+    if (ret == 0) {
+        ret = tsipImportPublicKey(tuc, tuc->wrappedKeyType);
+    }
+
+    if (ret == 0) {
+        if (isRsa) {
+            if (!tuc->ClientRsa2048PubKey_set) {
+                ret = NO_PRIVATE_KEY;
+            }
+        }
+        else {
+#if defined(WOLFSSL_CHECK_SIG_FAULTS)    
+            if (!tuc->ClientEccP256PubKey_set) {
+                ret = NO_PRIVATE_KEY;
+            }
+#endif /* WOLFSSL_CHECK_SIG_FAULTS */
+        }
+    }
+
+    if (ret == 0) {
+        sigData = tuc->sigDataCertVerify;
+
+        idx = 0;
+        ForceZero(sigData, sizeof(tuc->sigDataCertVerify));
+        XMEMSET(sigData, TSIP_SIGNING_DATA_PREFIX_BYTE, 
+                                                TSIP_SIGNING_DATA_PREFIX_SZ);
+
+        idx += TSIP_SIGNING_DATA_PREFIX_SZ;
+        XMEMCPY(&sigData[idx], clientCertVfyLabel, TSIP_CERT_VFY_LABEL_SZ);
+
+        idx += TSIP_CERT_VFY_LABEL_SZ;
+        XMEMCPY(&sigData[idx], hash, hashSz);
+    }
+
+    if (ret == 0) {
+        /* extract signature data from generated CertificateVerify message */
+        if (isRsa) {
```

Comment:
Should this `if (isRsa)` be empty?  If this is intended, we should refactor to put the else logic inside an `if (!isRsa)` or similar.

Reply:
Changed to use  `if (!isRsa)`  to remove empty block.